hostinfo/README.md

```diff
@@ -23,15 +23,14 @@ easier than creating and integrating a normal check into the rackspace monitorin
 
 There's a few ways to run hostinfo checks on a remote box.  
 
-1) Through the monitoring system, remotely, you can make a curl request:
+1) Through the monitoring system, remotely. You can make a curl request:
 ```sh
 curl -H 'X-Auth-Token: <auth token>' -H 'X-Tenant-Id: <tenant id>'  https://monitoring.api.rackspacecloud.com/v1.0/agents/<agent_id>/host_info/<hostinfo_type>
 ```
 Of course the above method can also be salvaged to programmatically allow usage from a user written script since all it  
  does is use the rackspace monitoring API.  
  
-2) SSHd into the box itself (Caveat: run these commands with sudo since the agent usually has super user privileges when  
-  run via the monitoring system).
+2) SSHd into the box itself (Caveat: run these commands with sudo since the agent usually has super user privileges when run via the monitoring system).
 ```sh
 rackspace-monitoring-agent -e hostinfo_runner -x <hostinfo type>
```

Comment:
I dislike even mentioning this as this is not a approved use case.  The -e option is solely for debugging in my opinion.